README.md

```diff
@@ -0,0 +1,25 @@
+# Home Assistant Yellow website
+
+This is the source of the Yellow documentation website at [yellow.home-assistant.io](https://yellow.home-assistant.io). 
+
+## Setup
+
+This website is built using [11ty](https://www.11ty.dev/). The website is automatically deployed using Netlify.
+
+## Development
+
+The easiest way to edit the documentation is to open the environment in Github Codespaces. Codespaces work in your browser, no installation required.
+1. Further up on this page, select the green **Code** button. 
+2. On the **Codespaces** tab, select **Open in codesapce**.
```

Comment:
```suggestion
2. On the **Codespaces** tab, select **Create codespace on main**.
```

Reply:
Presumably I'm getting this button text since I've never used Codespaces for this repo before